pages/api/posts/index.js

```diff
@@ -6,7 +6,18 @@ import handleError from '../../../utils/handleError'
 const handler = async (req, res) => {
     if (req.method === 'GET') {
         try {
-            const posts = await Post.find({}).sort({ _id: -1 }) //sorts by date newest to oldest
+            let { page, size } = req.query
+            if (!page) {
+                page = 1
+            }
+            if (!size) {
+                size = 10
+            }
```

Comment:
change this to 
```typescript
const { page = 1, size = 10 } = req.query 
```
its a cleaner way to have default values, and prevents the need for `let`